src/item_menu.c

```diff
@@ -2173,19 +2175,19 @@ void Task_WallyTutorialBagMenu(u8 taskId)
 
     if (!gPaletteFade.active)
     {
-        switch (data[8])
+        switch (tItemCount)
```

Comment:
`tItemCount` is not appropriate here.  `data[8]` is just some state tracker in this context.